packages/ndla-ui/src/Gloss/Gloss.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import styled from '@emotion/styled';
+import { colors, spacing, misc } from '@ndla/core';
+import { AccordionRoot, AccordionItem, AccordionHeader, AccordionContent } from '@ndla/accordion';
+import SpeechControl from '../AudioPlayer/SpeechControl';
+
+interface Example {
+  example: string;
+  language: string;
+}
+
+interface Alternatives {
+  traditionalChinese?: string;
+  pinyin?: string;
+  norwegianTranslation: string;
+}
```

Comment:
Trur eg heller ville hatt noe som ligner på:
```
type Alternative = 'traditional' | 'pinyin';
interface Transcription {
  alternative: Alternative;
  value: string;
}
```
For ordet i seg sjølv har title på originalspråket.